src/ted2zim/entrypoint.py

```diff
@@ -189,8 +194,8 @@ def main():
     from ted2zim.scraper import Ted2Zim
 
     try:
-        if args.topics and args.playlist:
-            parser.error("--topics is incompatible with --playlist")
+        if sum(1 for x in [args.topics, args.playlist, args.link] if x) > 1:
```

Comment:
Please use argparse mutually exclusive groups, it will be more obvious to read : https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group